src/pages-and-resources/discussions/app-config-form/apps/shared/messages.js

```diff
@@ -30,6 +30,16 @@ const messages = defineMessages({
     id: 'authoring.discussions.builtIn.divideCourseWideTopics.help',
     defaultMessage: 'Choose which of your general course wide discussion topics you would like to divide.',
   },
+  divideGeneralTopic: {
+    id: 'authoring.discussions.builtIn.divideGeneralTopic.label',
+    defaultMessage: 'General',
+    description: 'Label for a checkbox allowing a user to divide the General course wide topic.',
+  },
+  divideQuestionsForTAsTopic: {
+    id: 'authoring.discussions.builtIn.divideQuestionsForTAsTopic.label',
+    defaultMessage: 'Questions for the TAs',
+    description: 'Label for a checkbox allowing a user to divide the Questions for the TAs course wide topic.',
```

Comment:
is TA a common acronym such that we expect everyone to know how to translate it? also looking at both of these out of context, I have no idea what it means to "divide a topic"

Reply:
same at L27, 31

Reply:
It's a bit confusing, as this is all sub-settings under "Divide discussions by cohorts" - visually it's understandable in the UI, but not so much here. I'll improve the description lines to reflect this.